apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function InstructorCourseAdminSettings({
+  resLocals,
+  needToSync,
+  noInfo,
+}: {
+  resLocals: Record<string, any>;
+  needToSync: boolean;
+  noInfo: boolean;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", { ...resLocals })}
+        <style>
+          .popover {
+            max-width: 50%;
+          }
+        </style>
+      </head>
+      <body>
+        <script>
+          $(function () {
+            $('[data-toggle="popover"]').popover({
+              sanitize: false,
+            });
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
+        <main id="content" class="container-fluid">
+          ${renderEjs(__filename, "<%- include('../partials/courseSyncErrorsAndWarnings'); %>", {
+            ...resLocals,
+          })}
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex">
+              Course ${resLocals.course.short_name}
```

Comment:
Nit, let's change this for consistency with all the other pages.

```suggestion
              Settings
```

Reply:
Changed locally in [91314d4](https://github.com/PrairieLearn/PrairieLearn/pull/9406/commits/91314d4c3e75e28d2bda3cfe367f712392d9d690).